MC-1438 Integrate MapBrowser into SQL Browser (#189)
hazelcast/management-center-docs: docs/modules/tools/pages/sql-browser.adoc

.Example query results
image:ROOT:SQLBrowserQueryResultTab.png[A list of query results]

> === Map browser

Reviewer: ```suggestion
=== Viewing Map Metadata
```

> If you run *simple SELECT queries (without JOIN and AGGREGATION)* over IMap Mappings then you can get access to xref:data-structures:map.adoc#map-browser[entry metadata].

Reviewer: ```suggestion
If you run simple `SELECT` queries without joins or aggregations over maps, you can view xref:data-structures:map.adoc#map-browser[metadata] for each entry that is returned by the query.
```

image:ROOT:SQLBrowserQueryResultTabWithClosedMapBrowser.png[A list of query results with buttons]

> By clicking on *button with chevron icon* at the beginning of each SQL result table row, metadata view opens below a row

Reviewer: ```suggestion
To view an entry's metadata, click the chevron (*⌄*) button next to a map entry in the query results.
```

> image:ROOT:SQLBrowserQueryResultTabWithOpenMapBrowser.png[A list of query results with metadata view]

Reviewer: This image displays a -1 for *Hits*. Is this expected?

Reviewer 2: it is a known bug on the member side, to be fixed before 5.2 release https://hazelcast.atlassian.net/browse/HZ-1313
@Sodik could you attach a screenshot with non-negative Hits value? it should work fine if there is a single member

.Example query results
image:ROOT:SQLBrowserQueryResultTab.png[A list of query results]

=== Map browser

> If you run *simple SELECT queries (without JOIN and AGGREGATION)* over IMap Mappings then you can get access to xref:data-structures:map.adoc#map-browser[query medata].

Reviewer 2: query medata -> entry metadata?

Author: changed

image:ROOT:SQLBrowserQueryResultTabWithClosedMapBrowser.png[A list of query results with chevron buttons]

> By clicking on *chevron button* at the beginning of each SQL result table row, metadata view opens below a row

Reviewer 2: is it `chevron button` or `chevron sign`?

Author: button with chevron icon

> image:ROOT:SQLBrowserQueryResultTabWithOpenMapBrowser.png[A list of query results with metada view]

Reviewer 2: metada -> metadata?

Author: done

> You can view results in two modes:

Reviewer 2: please remove the information about modes or state that they are applicable only to pre-5.2 clusters

Author: removed
